maintenance_plan/__manifest__.py

```diff
@@ -3,23 +3,24 @@
 {
     "name": "Maintenance Plan",
     "summary": "Extends preventive maintenance planning",
-    "version": "16.0.1.0.0",
+    "version": "17.0.1.0.0",
     "author": "Camptocamp SA, ForgeFlow, Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "category": "Maintenance",
     "website": "https://github.com/OCA/maintenance",
     "images": [],
-    "depends": ["base_maintenance"],
+    "depends": ["base_maintenance", "maintenance"],
```

Comment:
I think maintenance is not required (base_maintenance depends on it)

https://github.com/OCA/maintenance/blob/17.0/base_maintenance/__manifest__.py#L12

Reply:
Sorry, my mistake.

Reply:
@etobella 
Done.